mget/request.go

```diff
@@ -0,0 +1,113 @@
+package os2
```

Comment:
Package name should be the same as the containing folder.

Reply:
Whoops, copy pasta got me.